fpga/Makefile

```diff
@@ -14,6 +14,9 @@ sim_name := none
 #########################################################################################
 # include shared variables
 #########################################################################################
+
+VIVADO_VERSION := $(shell vivado -version | head -1 | grep -o -P '(?<=\s).*(?=\s)')
```

Comment:
If Vivado isn't on the path this pollutes the terminal output. Maybe pipe stderr to `/dev/null`. Also, where is this checked?